### PR TITLE
Fix breaking change

### PR DESCRIPTION
**Pull request by mit-mit**
Update backgroundColor -> color to match https://github.com/flutter/flutter/pull/9648

### CODE REVIEW

**Comment by mit-mit** [comment]
Fixes travis failure:

https://travis-ci.org/flutter/website/builds/226718731

```
Analyzing /home/travis/build/flutter/website/_includes/_code/container/main.dart...
  warning • The named parameter 'backgroundColor' isn't defined at _includes/_code/container/main.dart:41:9 • undefined_named_parameter
1 issue found.
(Ran in 5.6s)
```

**Comment by sethladd** [comment]
Looks like we need to update backgroundImage, too.

**Comment by sethladd** [comment]
lgtm

File: _includes/_code/container/main.dart [hunk 1]
```diff
@@ -38,7 +38,7 @@ class _MyHomePageState extends State<MyHomePage> {
   Widget build(BuildContext context) {
     var container = new Container(
       decoration: new BoxDecoration(
-        backgroundColor: Colors.black26,
+        color: Colors.black26,
```
Comment (Hixie) [inline _includes/_code/container/main.dart]: For cases like this where there's a BoxDecoration with nothing but a `color`, in a Container, you can replace the `decoration` property with a `color` property that just takes the Color and avoid the BoxDecoration entirely.

Comment (Hixie) [inline _includes/_code/container/main.dart]: This applies to most cases here.

**Pull request merged**